app/api/survey/route.ts

```diff
@@ -19,13 +21,13 @@ export async function GET() {
         const records = await base(tableId)
         .select({})
         .all();
-        
-        const formattedRecords = records.map(record => ({
-        id: record.id,
-        ...record.fields
+
+        const results = records.map(record => ({
+          ...record.fields as SurveyResults,
```

Comment:
We can avoid casting by passing a generic type into Airtable - 

```suggestion
        const records = await base<SurveyResults>(tableId).select().all();

        const results = records.map(({ fields }) => fields);
```

Under the hood I guess it's still "casting" somewhere, but this is a bit neater and more accurate!

Note: Looks like we need to swap out `null` for `undefined` in the `SurveyResults` type 👍